packages/shared/src/features/shortcuts/components/modals/MostVisitedSitesModal.tsx

```diff
@@ -0,0 +1,69 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import { LazyImage } from '../../../../components/LazyImage';
+import { Button, ButtonVariant } from '../../../../components/buttons/Button';
+import type { ModalProps } from '../../../../components/modals/common/Modal';
+import { Modal } from '../../../../components/modals/common/Modal';
+import { Justify } from '../../../../components/utilities';
+import { useShortcutLinks } from '../../hooks/useShortcutLinks';
+import { useShortcuts } from '../../contexts/ShortcutsProvider';
+
+export function MostVisitedSitesModal({
+  className,
+  ...props
+}: ModalProps): ReactElement {
+  const { askTopSitesBrowserPermission } = useShortcutLinks();
+  const { setIsManual, setShowPermissionsModal } = useShortcuts();
+
+  const onRequestClose = () => {
+    setShowPermissionsModal(false);
+    setIsManual(true);
+  };
+
+  return (
+    <Modal
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.Medium}
+      {...props}
+      onRequestClose={onRequestClose}
+    >
+      <Modal.Header />
+      <Modal.Body>
+        <Modal.Title className="mb-4">Show most visited sites</Modal.Title>
```

Comment:
Why don't you just pass the title to modalHeader? it should have a title attribute

Reply:
Title appears after children. So I removed it from prop and added it here. But could probably just use css to re-order it

Reply:
Ah, this was like that previosuly